src/app/shared/custom-components/offer-item/offer-item.component.scss

```diff
@@ -1,135 +1,157 @@
 @import '/src/app/styles/variables.scss';
-
-:host::ng-deep .mat-card-header-text {
-  margin: 0.6em;
-}
+$small-card: 1500px;
 
 .offer-card {
+  position: relative;
   display: flex;
-  margin-bottom: 0.5em;
   align-items: center;
-  justify-content: space-between;
-  color: #383c3d;
-  padding-top: 1.5em;
-
-  &__image {
-    display: inline-block;
-    width: 100px;
-    object-fit: cover;
-    height: 75px;
-    border-radius: 0%;
-  }
+  color: $text-primary;
+  margin-bottom: 0.5em;
 
-  &__discount {
+  &__link-overlay {
+    position: absolute;
     top: 0;
+    left: 0;
     right: 0;
-    position: absolute;
-    background-color: rgba($color: $color-discount, $alpha: 0.3);
-    color: $color-discount;
-    min-width: 50px;
-    padding: 0.2em 1em;
-    font-size: 18px;
-    font-weight: 500;
-    border-bottom-left-radius: 0.5em;
+    bottom: 0;
+    z-index: 1;
   }
 
-  &__properties {
-    align-self: flex-end;
-    color: $background-icon;
-    position: relative;
-    top: 0.5em;
+  &__content {
+    display: flex;
   }
 
-  &__statistics {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
+  &__image {
+    object-fit: cover;
+    width: 20%;
+    margin-right: 1em;
+
+    @media screen and (max-width: $small-card) {
+      max-width: 120px;
+    }
+
+    @media screen and (max-width: $mobile) {
+      display: none;
+    }
   }
 
-  &__views,
-  &__using {
+  &__text {
     display: flex;
-    align-items: center;
+    flex-flow: column;
     justify-content: space-between;
 
-    > mat-icon {
-      margin-right: 0.1em;
+    @media screen and (max-width: $small-card) {
+      position: relative;
+      top: 0.3em;
     }
   }
-}
-
-.offer-card-header {
-  max-width: 600px;
 
   &__title {
     font-size: 16px;
     line-height: 24px;
+
+    // Limiting text to 2 lines
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    -webkit-line-clamp: 2;
   }
 
-  &__subtitle {
-    margin-bottom: 0;
+  &__vendor {
+    color: $text-secondary;
   }
-}
 
-.stars {
-  width: 120px;
-  text-align: center;
-  color: $blue-primary;
+  &__properties {
+    position: relative;
+    top: 1em;
 
-  > mat-icon {
-    width: 24px;
-    height: 24px;
-    font-size: 24px;
-  }
-}
+    @media screen and (max-width: $small-card) {
+      display: flex;
+      align-items: flex-start;
+      width: auto;
+      top: 0em;
+    }
 
-@media screen and (max-width: $desktop) {
-  :host::ng-deep .mat-card-header-text {
-    margin: 0.3em;
+    @media screen and (max-width: $mobile) {
+      top: 0.3em;
+    }
   }
 
-  .offer-card {
-    &__image {
-      width: 75px;
-      height: 55px;
-    }
+  &__statistics {
+    display: flex;
+    justify-content: space-around;
+    color: $background-icon;
 
-    &__discount {
-      font-size: 16px;
+    @media screen and (max-width: $small-card) {
+      order: -1;
```

Comment:
You should leave a comment. I don't understand what it has to do

Reply:
ok